integration-tests/spring-web/src/test/java/io/quarkus/it/spring/web/SpringCacheControllerTest.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.it.spring.web;
+
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.restassured.RestAssured;
+
+@QuarkusTest
+public class SpringCacheControllerTest {
+
+    @Test
+    public void testCache() {
+        // first invocation
+        RestAssured.when().get("/cache/greet/george").then()
+                .contentType("application/json")
+                .body(containsString("george"), containsString("0"));
+
+        // second invocation should yield same count
+        RestAssured.when().get("/cache/greet/george").then()
+                .contentType("application/json")
+                .body(containsString("george"), containsString("0"));
+
+        // invocation with different key should yield different count
+        RestAssured.when().get("/cache/greet/geo").then()
+                .contentType("application/json")
+                .body(containsString("geo"), containsString("1"));
```

Comment:
Isn't that risky to look for `geo` here while it could match both `geo` and `george`?

Reply:
That's a good idea, fixed now